docs/src/content/docs/environmental-impact.md

```diff
@@ -113,6 +114,7 @@ These tests with the [Website Carbon Calculator][wcc] compare similar pages buil
 [dy-carbon]: https://www.websitecarbon.com/website/docsify-js-org/
 [ds-carbon]: https://www.websitecarbon.com/website/docusaurus-io-docs/
 [rtd-carbon]: https://www.websitecarbon.com/website/docs-readthedocs-io-en-stable-index-html/
+[rp-carbon]: https://www.websitecarbon.com/website/rspress-dev-guide-start-introduction-html/
```

Comment:
This is an unfair comparison, the Rspress introduction page contains some images.

Reply:
Oops, you are correct; I should have tested other pages before creating the PR.

Reference:
https://www.websitecarbon.com/website/rspress-dev-guide-start-getting-started-html/

Reply:
Thank you for your interest in Rspress😄. We will optimize the image size on the Rspress introduction page and continuously reduce the carbon metrics for all Rspress users.